package.json

```diff
@@ -17,6 +17,8 @@
     "framer-motion": "^11.3.30",
     "lodash.debounce": "^4.0.8",
     "lodash.throttle": "^4.1.1",
+    "@deck.gl/widgets": "^9.0.16",
```

Comment:
Nit: can we alphabetically sort our dependencies? I'd like to visually keep track of deck.gl dependencies next to each other.

Also, all deck.gl libraries should have the same version constraint.